Applications/MIDIMonitor/SMMAppController.m

```diff
@@ -332,9 +331,21 @@ - (void)failedToConnectToSpyClient
 
 - (void)sourceEndpointsAppeared:(NSNotification *)notification
 {
-    if ([[NSUserDefaults standardUserDefaults] boolForKey:SMMOpenWindowsForNewSourcesPreferenceKey]) {
+    NSInteger option = [[NSUserDefaults standardUserDefaults] integerForKey:SMMAutoConnectNewSourcesPreferenceKey];
+    
+    if (option == 1) {
```

Comment:
Please make an enum for these values somewhere, instead of using magic numbers 1 and 2.